streamwebs_frontend/streamwebs/models.py

```diff
@@ -77,21 +77,107 @@ def __unicode__(self):
         return self.user.username
 
 
+class WaterQualityManager(models.Manager):
+    """
+    Manager for the Water_Quality model/datasheet.
+    """
+    def create_water_quality(self, site, date, school, DEQ_dq_level=None,
+                             latitude=0, longitude=0, fish_present=None,
+                             live_fish=0, dead_fish=0, air_temp_unit=None,
+                             water_temp_unit=None, notes=''):
```

Comment:
It might be better not to provide defaults in your manager for required fields, since you have written tests that are all about testing object creation with required fields vs. with required fields and optional fields. This means that in your tests, whenever you use this manager, you will have to provide arguments for every field except notes.